### PR TITLE
[stable/nginx-ingress] Bump nginx-ingress version to 0.29.0

### PR DESCRIPTION
**Pull request by Vlatombe**
#### What this PR does / why we need it:
Bump nginx-ingress version to 0.29.0

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/nginx-ingress/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.30.3
-appVersion: 0.28.0
+version: 1.30.4
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/Chart.yaml]: Let's bump the minor version.
`1.31.0`

Comment (Vlatombe) [reply stable/nginx-ingress/Chart.yaml]: Done

**Comment by ChiefAlexander** [comment]
/lgtm
Thanks!

**Pull request merged**